private/append_common.m

```diff
@@ -131,6 +131,13 @@
             chansel = match_str(varargin{j}.label, oldlabel{j});
             data.(cfg.parameter{i})(:,chansel,chansel) = varargin{j}.(cfg.parameter{i})(:,chansel,chansel);
           end
+         
+        case {'chan_freq_time'}
+          data.(cfg.parameter{i}) = nan(dimsiz);
+          for j=1:numel(varargin)
+            chansel = match_str(varargin{j}.label, oldlabel{j});
+            data.(cfg.parameter{i})(chansel,:,:) = varargin{j}.(cfg.parameter{i})(chansel,:,:);
```

Comment:
I wonder whether this works generically, i.e. I think this should work also, when the dimord = 'chan', 'chan_freq', or 'chan_time'.